### PR TITLE
Update 'About BTBB' in User Guide

### PR DESCRIPTION
**Pull request by EuniceSim142**
Resolves part of #169.
[User Guide](https://eunicesim142.github.io/tp/UserGuide.html)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -18,11 +18,21 @@ Reused from https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/179/commits/aec461
...
+Can't even tell if you're making a profit? Don't worry. <br>
+Have agile fingers and want to show it off? Weird flex but ok! <br>
+
+BTBB will do all these and MORE, just for you!
+
+BTBB is a desktop application that aims to help home-based F&B businesses, by keeping track of logistics as well as
+providing valuable data visualizations, via a user-friendly command-line interface (CLI). Our
 application centralizes all data in one place, and even comes with a Graphical User Interface (GUI) to easily view and
-manoeuvre through client and order details. If you are looking for an easy solution to manage your business, give BTBB a try!
+manoeuvre through client and order details.
```
Comment (sivayogasubramanian) [inline docs/UserGuide.md:35]: 😀

File: docs/UserGuide.md [hunk 2]
```diff
@@ -18,11 +18,21 @@ Reused from https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/179/commits/aec461
...
-providing valuable data visualizations, via a user-friendly command-line interface (CLI).** Keeping track of
-information from inventory to order records for delivery can be a hassle if you are a one-man show. That's why, our
+Can you prepare the most heavenly dishes? Why not share the love (and earn a little extra cash on the side :stuck_out_tongue_closed_eyes:) with **BobTheBistroBoss**! <br>
+BobTheBistroBoss (BTBB) supports Home Chefs who want to venture into the home-based F&B business, but are deterred by the complexity of business management. <br>
+BTBB makes this process **hassle-free**! Allow us to handle the business, you just focus on being the best chef you can be!
+
+Too many orders? No problem. <br>
+Ingredients always running out? That's OK! <br>
+Can't even tell if you're making a profit? Don't worry. <br>
+Have agile fingers and want to show it off? Weird flex but ok! <br>
```
Comment (Manoharan-Ajay-Anand) [inline docs/UserGuide.md:28]: I don' think need this line. Seems weird

File: docs/UserGuide.md [hunk 3]
```diff
@@ -18,11 +18,21 @@ Reused from https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/179/commits/aec461
 
 ### 1.1 About BobTheBistroBoss
 
-BobTheBistroBoss (BTBB) is a **desktop application that aims to help home-based F&B businesses, by keeping track of logistics as well as
-providing valuable data visualizations, via a user-friendly command-line interface (CLI).** Keeping track of
-information from inventory to order records for delivery can be a hassle if you are a one-man show. That's why, our
+Can you prepare the most heavenly dishes? Why not share the love (and earn a little extra cash on the side :stuck_out_tongue_closed_eyes:) with **BobTheBistroBoss**! <br>
+BobTheBistroBoss (BTBB) supports Home Chefs who want to venture into the home-based F&B business, but are deterred by the complexity of business management. <br>
+BTBB makes this process **hassle-free**! Allow us to handle the business, you just focus on being the best chef you can be!
```
Comment (Manoharan-Ajay-Anand) [inline docs/UserGuide.md:23]: How about "Allow us to handle the business, you just focus on being the next Gordon Ramsay".

**Review by Manoharan-Ajay-Anand — COMMENTED** [review]
Nice Intro

File: docs/UserGuide.md [hunk 4]
```diff
@@ -18,11 +18,19 @@ Reused from https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/179/commits/aec461
...
-information from inventory to order records for delivery can be a hassle if you are a one-man show. That's why, our
-application centralizes all data in one place, and even comes with a Graphical User Interface (GUI) to easily view and
-manoeuvre through client and order details. If you are looking for an easy solution to manage your business, give BTBB a try!
+Do you face difficulties managing many different orders? <br>
+Ever wanted to stop worrying about your next inventory restock? <br>
+Thought about hiring someone to manage the growth of your home business? <br>
+
+If any of these concerns apply to you, then you're in luck –– **BobTheBistroBoss** can do all these and MORE! <br>
+Not only that, BobTheBistroBoss (BTBB) makes this process **hassle-free**! <br>
+Allow BTBB to handle the business, you just focus on being the next Gordon Ramsay!
```
Comment (zognin) [inline docs/UserGuide.md:27]:
```suggestion
Allow BTBB to handle the business, while you focus on being the next Gordon Ramsay!
```

**Review by zognin — COMMENTED** [review]
Intro is very nice! LGTM with one minor nit

File: docs/UserGuide.md [hunk 5]
```diff
@@ -3,28 +3,23 @@ layout: page
...
 
-### 1.1 About BobTheBistroBoss
+BTBB supports Home Chefs who seek to **venture into the home-based F&B business**, but are deterred by the complexity of business management.
 
-BobTheBistroBoss (BTBB) is a **desktop application that aims to help home-based F&B businesses, by keeping track of logistics as well as
-providing valuable data visualizations, via a user-friendly command-line interface (CLI).** Keeping track of
-information from inventory to order records for delivery can be a hassle if you are a one-man show. That's why, our
-application centralizes all data in one place, and even comes with a Graphical User Interface (GUI) to easily view and
-manoeuvre through client and order details. If you are looking for an easy solution to manage your business, give BTBB a try!
+With BTBB, you can track **logistics** and receive valuable **data visualizations**, via a **user-friendly** command line interface (CLI).
```
Comment (zognin) [inline docs/UserGuide.md:18]: I think this comma can be removed?
```suggestion
With BTBB, you can track **logistics** and receive valuable **data visualizations** via a **user-friendly** command line interface (CLI).
```

**Review by zognin — APPROVED** [review]
LGTM one minor comment

**Pull request merged**